<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>repSklad</class>
 <widget class="QWidget" name="repSklad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отчет по складу</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Параметры отчета</string>
   </property>
   <widget class="QDateLineEdit" name="date_start">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата начала:</string>
    </property>
   </widget>
   <widget class="QDateLineEdit" name="date_end">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата окончания:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lOtdel_Sklada">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Отдел склада:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="otdel_skalda">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="start_report">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>132</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сформировать</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDateLineEdit</class>
   <extends>QWidget</extends>
   <header>qdatelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
